### PR TITLE
DOC: Fix return argument readable in ICBM fetcher doc string

### LINKED ISSUE #1166: nilearn.datasets.fetch_icbm152_2009 docstring isn't great
The description of the return argument in nilearn.datasets.fetch_icbm152_2009 is hard to read.

Also, we should add a seealso block to point to nilearn.datasets.load_mni152_template and fetch_icbm152_brain_gm_mask

### PR DESCRIPTION
**Pull request by KamalakerDadi**
Closes #1166  .

Changes proposed in this pull request:

- Improve return argument doc string of function `fetch_icbm152_2009`
- Added related loaders and fetchers to See Also

### CODE REVIEW

**Review by NicolasGensollen — APPROVED** [review]
LGTM, thanks @KamalakerDadi !

File: nilearn/datasets/struct.py [hunk 1]
```diff
@@ -54,13 +54,47 @@ def fetch_icbm152_2009(data_dir=None, url=None, resume=True, verbose=1):
...
-        "t1", "t2", "t2_relax", "pd": anatomical images obtained with the
-        given modality (resp. T1, T2, T2 relaxometry and proton
-        density weighted). Values are file paths.
-        "gm", "wm", "csf": segmented images, giving resp. grey matter,
-        white matter and cerebrospinal fluid. Values are file paths.
-        "eye_mask", "face_mask", "mask": use these images to mask out
-        parts of mri images. Values are file paths.
+
+        - "t1": str,
+          Path to anatomical image obtained with the T1 weighted
```
Comment (bthirion) [inline nilearn/datasets/struct.py:59]:
```suggestion
          Path to T1-weighted anatomical image
```

File: nilearn/datasets/struct.py [hunk 2]
```diff
@@ -54,13 +54,47 @@ def fetch_icbm152_2009(data_dir=None, url=None, resume=True, verbose=1):
...
+        - "t2": str,
+          Path to anatomical image obtained with the T2 weighted
```
Comment (bthirion) [inline nilearn/datasets/struct.py:61]:
```suggestion
          Path to the T2-weighted anatomical image
```

File: nilearn/datasets/struct.py [hunk 3]
```diff
@@ -54,13 +54,47 @@ def fetch_icbm152_2009(data_dir=None, url=None, resume=True, verbose=1):
...
+        - "t2_relax": str,
+          Path to anatomical image obtained with the T2 relaxometry
+        - "pd": str,
+          Path to anatomical image obtained with the proton density
```
Comment (bthirion) [inline nilearn/datasets/struct.py:65]:
```suggestion
          Path to the proton density weighted anatomical image
```

File: nilearn/datasets/struct.py [hunk 4]
```diff
@@ -54,13 +54,47 @@ def fetch_icbm152_2009(data_dir=None, url=None, resume=True, verbose=1):
...
+          weighted
+        - "gm": str,
+          Path to grey matter segmented image
+        - "wm": str,
+          Path to white matter segmented image
+        - "csf": str,
+          Path to cerebrospinal fluid segmented image
+        - "eye_mask": str,
+          Path to eye mask useful to mask out part of mri images
```
Comment (bthirion) [inline nilearn/datasets/struct.py:74]:
```suggestion
          Path to eye mask useful to mask out part of MRI images
```

File: nilearn/datasets/struct.py [hunk 5]
```diff
@@ -54,13 +54,47 @@ def fetch_icbm152_2009(data_dir=None, url=None, resume=True, verbose=1):
...
+        - "face_mask": str,
+          Path to face mask useful to mask out part of mri images
```
Comment (bthirion) [inline nilearn/datasets/struct.py:76]:
```suggestion
          Path to face mask useful to mask out part of MRI images
```

**Review by bthirion — COMMENTED** [review]
LGTM, thx !
Just proposing some docstring improvements.

**Review by bthirion — APPROVED** [review]
LGTM, thx !

**Comment by NicolasGensollen** [comment]
Thanks, merging....

**Pull request merged**